src/ape/exceptions.py

```diff
@@ -1,10 +1,13 @@
+from __future__ import annotations
```

Comment:
Is this necessary to do?

Reply:
Still WIP, something wasn't working with a type and trying to place it under the `if TYPING` so I am just bypassing it for now until the PR is ready